### DISCUSSION TITLE
Any QScroller for QScrollArea ??

### DISCUSSION THREAD

**Discussion by cyfung1031** [quasarframework/quasar#10715] [General - Components / Directives / etc] [answered]
For QPageScroller,  it is used for QPage to scroll back to the top
Is there any similar component for QScrollArea?

**Comment by ontwikkelfabriek** [comment, marked as answer]
@cyfung1031 

> Is there any similar component for QScrollArea?

Not as easily like QPagescroller, but you can implement the same behaviour with a Q-page-sticky. A combination of the following examples.
https://quasar.dev/layout/page-sticky#basic
https://quasar.dev/vue-components/scroll-area#scroll-position

**Reply by cyfung1031** [reply]
could you give me a single example showing how can the Q-page-sticky used inside the QScrollArea?
both QPageSticky and QPageScroller are for QPage and I cannot found any similar thing for just QScrollArea
I cannot find such an example from these two links

**Reply by ontwikkelfabriek** [reply]
@cyfung1031 

You're right it's for q-page. 

I am not a css god, but I guess the first step to simulate something similiar, is to make a q-button float on top of it's parent ( the scroll-area)...

If you can accomplish that then the logic to scroll up will not be very hard.


Here's a (visual) start codepen, button is not  floating:
https://codepen.io/ontwikkelfabriek/pen/ExXQmgg

**Reply by cyfung1031** [reply]
> @cyfung1031
> 
> You're right it's for q-page.
> 
> I am not a css god, but I guess the first step to simulate something similiar, is to make a q-button float on top of it's parent ( the scroll-area)...
> 
> If you can accomplish that then the logic to scroll up will not be very hard.
> 
> Here's a (visual) start codepen, button is not floating:
> https://codepen.io/ontwikkelfabriek/pen/ExXQmgg

okay. I see what you mean. 
I just found out how to do it. Thanks.

**Reply by cyfung1031** [reply]
Remarks:
I have created the feature request for QScroller in #10716
The example coding of the combination can be found at [here](https://codepen.io/cyfung1031/pen/abwqLoN)
